R/set_rgcca.R

```diff
@@ -1,141 +1,80 @@
-# inds : individuals removed from the blocks (for crossvalidation)
-# affects same parameters as rgcca_res (or other ones if specified) on a subset 
-# of individuals determined by inds (individuals to remove)
-set_rgcca <- function(
-    rgcca_res,
-    blocks = NULL,
-    connection = NULL,
-    tau = 1,
-    sparsity = 1,

```

Comment:
I do not understand why `sparsity` and `tau` are by default set to `1`. Previously, is `block` was `NULL`, this default value was used instead of the ones in `rgcca_res$call`. I decided to set them by default as `NULL` as for the orther paramters and to check if they are `NULL`. If this is the case, I use their value in `rgcca_res$call`.